test-case-reporting/types/test-case-reporting.d.ts

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+declare module 'test-case-reporting' {
+  /**
+   * A standard logging interface.
+   */
+  export interface Logger {
+    debug(message: string, ...extraInfo: unknown[]): void;
+    warn(message: string, ...extraInfo: unknown[]): void;
+    error(message: string, ...extraInfo: unknown[]): void;
+    info(message: string, ...extraInfo: unknown[]): void;
+  }
+
+  /**
+   * Possible job types.
+   */
+  export type JobType = 'unit' | 'integration';
```

Comment:
When possible, try to avoid comments that don't add useful information. Here, something like:
`/** Travis job types for which test results may be reported. */`

The test status type below likewise could be slightly more descriptive, a la
`/** Possible statuses for a given test run. */`

Additional style nit: When the JSDoc is just one line, we can collapse it as shown above

Reply:
Done.